.azure/pipelines/ci.yml

```diff
@@ -334,12 +338,32 @@ jobs:
     jobDisplayName: "Test: macOS 10.13"
     agentOs: macOs
     buildScript: ./eng/scripts/cibuild.sh
-    buildArgs: --test
+    buildArgs: --test -p:RunTemplateTests=false
+    beforeBuild:
```

Comment:
If you left template tests in  "Test: Ubuntu 16.04 x64" and "Test: macOS 10.13" rather than splitting them up, how long would it take to finish the test run? If it isn't slower than 70 minutes each, it's probably not a good idea to split it up because the longest running job is still longer than these smaller parts. Maybe in the future, if we get our build fast enough, we can do this optimization?